examples/text_to_image/train_text_to_image_sdxl.py

```diff
@@ -325,6 +325,55 @@ def parse_args(input_args=None):
     parser.add_argument(
         "--lr_warmup_steps", type=int, default=500, help="Number of steps for the warmup in the lr scheduler."
     )
+    parser.add_argument(
+        "--timestep_bias_strategy",
+        type=str,
+        default="none",
+        choices=["earlier", "later", "none"],
+        help=(
+            "The timestep bias strategy, which may help direct the model toward learning low or frequency details."
```

Comment:
"low or frequency details" sounds a bit off to me.

Reply:
ah yes, this should have said "or high"